### DISCUSSION TITLE
ISO date format does not work for dateColumn

### DISCUSSION THREAD

**Discussion by maakle** [nick-keller/react-datasheet-grid#213] [Feature requests]
I am using the dateColumn like below:

```  
{
    ...keyColumn("remoteCreatedAt", dateColumn),
    title: "Created At",
  },
```

This is the value coming from my database: `remoteCreatedAt: "2022-11-07T10:37:58.000Z"` 
Instead of showing the date + date picker, it breaks the App with the following error:

```
Unhandled Runtime Error
TypeError: rowData.toISOString is not a function

Call Stack
eval
node_modules/react-datasheet-grid/dist/columns/dateColumn.js (47:0)
```

How do I fix this without just creating an own DateCell, grabbing the rowData, formatting it and handling it myself? Or anything I am doing wrong?

**Comment by nick-keller** [comment]
Hey @mathiasklenk ,
The built in date column is 57 lines of code.
You should be able to copy paste it and adapt it to your needs. Let me know if you are struggling.

**Comment by nick-keller** [comment]
Hey @mathiasklenk ,
This is now available in 4.8.0 as `isoDateColumn`

**Reply by maakle** [reply]
Great. Thank you!

**Reply by Y-Q** [reply]
Hi @nick-keller I tested the package and it works great for the most part. I did notice the isoDateColumn can't be copied/posted, or dragged. Is that something known?
